{"url": "https://api.github.com/repos/tensorflow/tensorflow/issues/comments/341925636", "html_url": "https://github.com/tensorflow/tensorflow/issues/2471#issuecomment-341925636", "issue_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/2471", "id": 341925636, "node_id": "MDEyOklzc3VlQ29tbWVudDM0MTkyNTYzNg==", "user": {"login": "zendelcrow", "id": 23545056, "node_id": "MDQ6VXNlcjIzNTQ1MDU2", "avatar_url": "https://avatars1.githubusercontent.com/u/23545056?v=4", "gravatar_id": "", "url": "https://api.github.com/users/zendelcrow", "html_url": "https://github.com/zendelcrow", "followers_url": "https://api.github.com/users/zendelcrow/followers", "following_url": "https://api.github.com/users/zendelcrow/following{/other_user}", "gists_url": "https://api.github.com/users/zendelcrow/gists{/gist_id}", "starred_url": "https://api.github.com/users/zendelcrow/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/zendelcrow/subscriptions", "organizations_url": "https://api.github.com/users/zendelcrow/orgs", "repos_url": "https://api.github.com/users/zendelcrow/repos", "events_url": "https://api.github.com/users/zendelcrow/events{/privacy}", "received_events_url": "https://api.github.com/users/zendelcrow/received_events", "type": "User", "site_admin": false}, "created_at": "2017-11-04T19:58:52Z", "updated_at": "2017-11-04T19:58:52Z", "author_association": "NONE", "body_html": "<p>Hi,</p>\n<p>The instrucction per_process_gpu_memory_fraction seems not working on my computer.</p>\n<p>I have Quadro K1200, 425/4039 MiB aprox, driver 387.12, cuda 8, cudnn 6,  tensorflow 1.4 install from anaconda. python 3.5.2 (64 bits).</p>\n<p>my code goes like this:<br>\nconfig = tf.ConfigProto()<br>\nconfig.gpu_options.per_process_gpu_memory_fraction = 0.4<br>\nwith tf.Graph().as_default():<br>\nwith tf.Session(config=config) as sess:<br>\n...</p>\n<p>And I get:</p>\n<p>2017-11-04 13:55:44.926218: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 3.94G (4235198464 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY<br>\n2017-11-04 13:55:44.933243: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 3.55G (3811678464 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY</p>\n<p>how can i check if I am running pywrap_tensorflow.list_devices()?</p>", "body_text": "Hi,\nThe instrucction per_process_gpu_memory_fraction seems not working on my computer.\nI have Quadro K1200, 425/4039 MiB aprox, driver 387.12, cuda 8, cudnn 6,  tensorflow 1.4 install from anaconda. python 3.5.2 (64 bits).\nmy code goes like this:\nconfig = tf.ConfigProto()\nconfig.gpu_options.per_process_gpu_memory_fraction = 0.4\nwith tf.Graph().as_default():\nwith tf.Session(config=config) as sess:\n...\nAnd I get:\n2017-11-04 13:55:44.926218: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 3.94G (4235198464 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY\n2017-11-04 13:55:44.933243: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 3.55G (3811678464 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY\nhow can i check if I am running pywrap_tensorflow.list_devices()?", "body": "Hi,\r\n\r\nThe instrucction per_process_gpu_memory_fraction seems not working on my computer. \r\n\r\nI have Quadro K1200, 425/4039 MiB aprox, driver 387.12, cuda 8, cudnn 6,  tensorflow 1.4 install from anaconda. python 3.5.2 (64 bits).\r\n\r\nmy code goes like this:\r\nconfig = tf.ConfigProto()\r\nconfig.gpu_options.per_process_gpu_memory_fraction = 0.4\r\nwith tf.Graph().as_default():\r\n        with tf.Session(config=config) as sess:\r\n            ...\r\n\r\n\r\nAnd I get:\r\n\r\n2017-11-04 13:55:44.926218: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 3.94G (4235198464 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY\r\n2017-11-04 13:55:44.933243: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 3.55G (3811678464 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY\r\n\r\nhow can i check if I am running pywrap_tensorflow.list_devices()?"}